docs/latest/modules/en/pages/use/security/self-signed-certificates.adoc

```diff
@@ -0,0 +1,371 @@
+= Self-signed certificates
+:description: SUSE Observability Self-hosted
+
+== SUSE Observability Server
+
+=== Overview
+
+SUSE Observability Server has several points of interaction with external systems. For example, event handlers can call out to webhooks in other systems. With the default configuration, SUSE Observability Server won't be able to communicate with these systems if they're secured with TLS using a self-signed certificate, or a certificate that isn't by default trusted by the JVM.
```

Comment:
this could use a bit more structure i think. As far as i see there are roughly two scenarios:
1. SUSE Observability needs to reach out to an external service with a self-signed certificate
2. SUSE Observability is itself behind a self-signed certificate. 
- Maybe oyu have another clearly demarked scenario?

I would start with these scenarios and guide the user to the one they have. I also wonder which one is more prevalent. With all our current air-gapped installations i feel scenario 2. actually might be the more important one, so we can put that as the first option

Reply:
The section for `SUSE Observability Server` was already there. For whatever reason, Github doesn't show it as renaming. 
For `SUSE Observability is itself behind a self-signed certificate` there are multiple options available depending on how users expose the server: what kind of ingress, load balancer, certificate management, etc.... This wasn't the scope of the task. If it is needed, we have to refine it again...

Reply:
I can, of course, add something generic for the platform behind self-signed certificate...